extensions/objects/templates/MathLatexStringEditorDirective.ts

```diff
@@ -16,6 +16,8 @@
  * @fileoverview Directive for math latex string editor.
  */
 
+require('directives/mathjax-bind.directive.ts');
```

Comment:
Missing import for `UrlInterpolationService`.

Reply:
Added.